code/modules/jobs/job_types/city/fixer/north.dm

```diff
@@ -13,8 +13,9 @@
 	display_order = JOB_DISPLAY_ORDER_FIXERLEAD
 	paycheck = 100
 	trusted_only = TRUE
-	access = list(ACCESS_XENOBIOLOGY, ACCESS_RC_ANNOUNCE) // Numero 55
+	access = list(ACCESS_XENOBIOLOGY, ACCESS_RC_ANNOUNCE)
 	minimal_access = list(ACCESS_XENOBIOLOGY, ACCESS_RC_ANNOUNCE)
+	departments = DEPARTMENT_COMMAND | DEPARTMENT_SECURITY
```

Comment:
Security is specifically for Lobotomy corp Agents, we actually originally put these under science

Reply:
fair, gonna revert dat then

Reply:
The reason for this is to give a distinction between LCorp and non-lcorp staff on maps that contain them both